src/Services/Logger.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Services;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+
+class Logger
+{
+    /**
+     * @var string
+     */
+    protected $logLevel = 'none';
```

Comment:
Can we have that `none` as a constant too? Maybe it make sense for it to be one of the log levels?